language/google/cloud/language/syntax.py

```diff
@@ -183,11 +185,17 @@ class Sentence(object):
     :param begin: The beginning offset of the sentence in the original
                   document according to the encoding type specified
                   in the API request.
+
+    :type sentiment: :class:`.Sentiment`
+    :param sentiment: For calls to `analyzeSentiment`_ or if
```

Comment:
I don't think this will actually resolve correctly, but I'm not 100% sure. Did it resolve correctly for you?